data/human/news.txt

```diff
@@ -2287,176 +2286,210 @@ news "Discontent"
-		"with substantial sections of hull heavily battered or missing altogether. Fair guess they didn't have enough anti-missle. "
-	word
-		"As it comes to a stop in the maitenance bay engineering shuttles begin "
+		"As it comes to a stop in the maintenance bay, the Bridi engineering shuttles begin "
 	word
 		"buzzing "
 		"fluttering "
 	word
 		"around it, assessing the damage and beginning the work on repairs. "
 	word
 		"The work won't take the shuttles long."
-		"They'll be working for the rest of the shift on this one."
 
 news "Symbol War"
 	location
-		planet Gagarin
-name
-	"Guerrilla Symbology"
-message
-	word
-		"It's an average day in a busy spaceport, "
-	word
-		"there's no real quiet anywhere in the port, "
-		"it's the expected amout of noisy crowd background noise, "
-		"with the normal buzz of activity with thousands of things happening every second, "
-	word
-		"it can be a bit much but anyone who works with ships learns to tune it out. "
-	word
-		"In spite of the drive to tune out the rumble of the crowds "
-		"However, against the drive to tune out the crowds "
-		"A rare and fleeting drop in the rumble allows% "
-	word
-		"you hear something that stands out "
-	word
-		"turning your attention to the unexpected noise, "
-		"glancing over to the new commotion, "
-	word
-		"you see a squad of "
-	word
-		"CCOR irregulars "
-		"CCOR guerrillas "
-	word
-		"a few of them are armed but most have paint, plaster, and ladders."
-	word
-		"They "
-	word
-		"leave the spaceport traffic alone,"
-		"aren't disrupting traffic,"
-	word
-		"rather "
-		"instead "
-	word
-		"they are "
-	word
-		"climbing the back wall toward the giant crest of the planetary government% "
-		"walking along the peripheral walls of the port focusing on the small government crests "
-		"returning to CCOR crests that have been covered with planetary government crests "
-	word
-		"they dilligently replace the government crests with CCOR's. "
-	word
-		"Spaceport security are "
-	word
-		"noticably absent."
-		"actively ignoring them."
-		"keeping their distance."
-	replace
-		"% you" " you to"
-		"% they dilligently replace the government crests" " they dilligently replace the large crest"
-		"%" ""
+		planet "Gagarin"
+	name
+		word
+			"Guerrilla Symbology"
+	message
+		word
+			"It's an average day in Gagarin's busy spaceport, until "
+		word
+			"your attention is grabbed by an oddity. A "
+			"an unusual commotion catches your attention. A "
+		word
+			"squad "
+			"band "
+			"team "
+		word
+			"of CCOR "
+		word
+			"guerillas "
+			"insurgents "
+		word
+			"have entered the port."
+		word
+			"Almost none "
+			"Few "
+			"Some "
+			"All "
+		word
+			"of them are armed, but it's their other equipment that stands out; paint, plaster, and ladders. They "
+		word
+			"scale the rear wall toward the giant crest of the local government and "
+			"walk along the walls, stopping at each local government crest and% "
```

Comment:
```suggestion
			"walk along the walls, stopping at each local government crest and %"
```
I don't think the replacement will work properly otherwise.